apps/i18n/poetry/en_us.json

```diff
@@ -43,6 +43,8 @@
   "noEffects": "You're missing some code. Add a background or text effect block under `when run`.",
   "noTextEffect": "Try adding a text color block under `when run`.",
   "oneTextEffect": "Great work! You added one text effect. Try adding another.",
+  "patLines": "I love to ride my skateboard all around\nWith my Walkman blasting, I don't hear a sound\nI just learned how to do a sweet new trick\nIt's called the kickflip and it's really sick\n\nSomething happening right now is Yo MTV Raps\nWatching Salt-N-Pepa and LL Cool J, mad snaps\nSlap bracelets are something cool\nAnd neon colors, they totally rule.",
```

Comment:
Do we know that this poem needs to be translated? I'm not sure what the translation plans are for CSC right now. If this poem is only meant for English, it could be defined in `poetry/constants.js` (alongside "My City") instead. I could imagine a string like this one could need some extra attention to correctly localize, so I wonder if it might be worth keeping it out of CrowdIn for now.

Reply:
Good question, asked in CSC. FWIW, most of the poetry poems are translated here, so I was following that pattern.

Reply:
Until now, all of the poems were for the Poem Art Hour of Code, which was designed to be friendly to international audiences. I'm just not sure if the same is true for CSC modules. The other poems in constants are ones that we didn't want to translate and/or only wanted to be available in certain languages.

Reply:
[discussion here](https://codedotorg.slack.com/archives/C02GRH2PYB1/p1677516711780519). This poem and one other will not be translated due to slang/proper names, all the others will be translated.